build.sh

```diff
@@ -56,9 +56,9 @@ CXX=$CXXCOMP CC=$CCOMP cmake . -DNO_NATIVE_OPTIMIZATION=1 -DCMAKE_BUILD_TYPE=${C
 make
 
 # Package Server
-mkdir Server/Licenses
-cp Install/ThirdPartyLicenses/* Server/Licenses/
+cp -r Install/ThirdPartyLicenses/ Server/
```

Comment:
Does this preserve the "licenses in a separate folder", or does it copy each license file to the Server folder directly?